Style Guide and Diataxis Review: Streams I (#212)
iotaledger/streams: documentation/docs/welcome.md

---

> description: IOTA Streams can be used for the integration of an IOTA Wallet into your application. 

Reviewer: ```suggestion
description: You can use IOTA Streams to integrate the IOTA Wallet into your application. 
```
